pkg/cmd/repo/clone/clone_test.go

```diff
@@ -168,6 +168,21 @@ func Test_RepoClone(t *testing.T) {
 			args: "Owner/Repo",
 			want: "git clone https://github.com/OWNER/REPO.git",
 		},
+		{
+			name: "Wiki with HTTPS URL",
+			args: "https://github.com/OWNER/REPO.wiki.git",
+			want: "git clone https://github.com/OWNER/REPO.git",
```

Comment:
This test expectation is wrong. The argument indicated that we want to clone the wiki. The URL that is used for clone does not have a `.wiki` suffix in it.